k2/csrc/compose.cu

```diff
@@ -370,7 +370,7 @@ class MultiGraphDenseIntersect {
                     as long as the number of active states in each FSA is
                     between min_active and max_active.
   */
-  Array1<float> GetPruningCutoffs(const Ragged<float> &arc_end_scores) {
+  Array1<float> GetPruningCutoffs(Ragged<float> &arc_end_scores) {
```

Comment:
Is there a constraint keeping from using `const` here?

Reply:
add documentation to say it is conceptually unchanged by this operation but we may update row_ids in its shape